lib/Model.php

```diff
@@ -571,7 +572,7 @@ public function attributes()
 	/**
 	 * Retrieve the primary key name.
 	 *
-	 * @param boolean Set to true to return the first value in the pk array only
+	 * @param boolean $first Set to true to return the first value in the pk array only
```

Comment:
The library itself is inconsistent on this. I didnt modify existing docblocks for grammar or style.